Swabbr.Core/Enums/ReactionState.cs

```diff
@@ -2,28 +2,40 @@
 
 namespace Swabbr.Core.Enums
 {
-
     /// <summary>
-    /// Represents the processing state of a reaction.
+    ///     Represents the processing state of a reaction.
     /// </summary>
     public enum ReactionState
     {
-
+        /// <summary>
+        ///     Reaction has been created without content.
+        /// </summary>
         [EnumMember(Value = "created")]
```

Comment:
`EnumMember` Should not be necessary

Reply:
Agreed, item already existed on my todo list.